### DISCUSSION TITLE
Improve Bots (2)

### DISCUSSION THREAD

**Discussion by 8BitAvenger** [Senryoku/Draftmancer#762] [General]
Hi, I've followed the previous 'Improve Bots' thread and tried my best to follow the linked dev work, but not at all studied/versed in dev/coding so idk what the heck happened, if progress was fruitful or not, if things were good and have recently reverted, or what.

I would love to put effort in any way I could to help improve the bots. In vintage cube I'm seeing bots pass 7th pick mox, time walk, ancestral, etc.

I, and I'm sure plenty of other players, would love an environment where bot drafting is at least somewhat realistic as there's something somewhat unique about an 8-person draft format, even if you don't have 8 people it can sometimes be the dream to reproduce as best as possible.

**Comment by Senryoku** [comment]
The bots used in cubes aren't always up-to-date with the latest cards. I handle this by falling back to way worse (but generic) bots when X% of cards aren't recognized. However, I was probably way too strict with the percentage, and more and more cubes were using the bad bots.

I recently - in the last month - soften the requirement a bit, so more drafts should use the good bots. Did you still have these issues in the last two weeks? If you did, could you share your exact cube list so I can investigate more?

**Reply by 8BitAvenger** [reply]
Oh, sweet! No, I haven't tried in the last couple weeks. That adds up to my experience, though. I was drafting LSVCube, very likely after a big 30+ card update. I'll try some more relatively soon and let you know if I still see pick 7 power, haha.
